modin/config/envvars.py

```diff
@@ -135,6 +135,26 @@ def _get_default(cls):
         return multiprocessing.cpu_count()
 
 
+class GpuCount(EnvironmentVariable, type=int):
+    """
+    How may GPU devices to utilize across the whole distribution
+    """
+
+    varname = "MODIN_GPUS"
+
+    # @classmethod
+    # def _get_default(cls):
+    #     import ray
+    #     if ray.is_initialized():
+    #         return len(ray.get_gpu_ids())
+    #     else:
+    #         print(
+    #             f"Ray has not been initialized yet. "
+    #             f"MODIN-GPU doesn't know the number of available GPUs. "
+    #         )
+    #         return 1
+
+
```

Comment:
Can you please comment wether these lines are needed or not?

Reply:
To decide the number of partitions, MODIN needs to know the number of available GPUs/CPUs. However, without initializing Ray, we cannot know the number of available GPUs/CPUs on a cluster. By commenting out these lines, we force the user to manually set this environment variable before using MODIN-GPU.

Reply:
Before we coming up with a better solution, I think we should force the user to set this environment variable manually before using MODIN-GPU. What do you think?